class/defaults.yml

```
@@ -33,3 +33,32 @@ parameters:
     generatedDefaultRoleBindingInNewNamespaces:
       bindingName: admin
       clusterRoleName: admin
+
+    generatedResourceQuota:
+      name: organization
+      synchronize: false
```

To my understanding, we shall prevent modifications altogether.
Consider the case where we want to adjust the quota for all namespaces, it should be enough to modify the cluster policy that takes care to synchronize all the existing quotas.
```suggestion
      synchronize: true
```
Yes, at some point we want to make them adjustable for special namespaces, but that's for later.